JAICore/jaicore-ml/src/main/java/ai/libs/jaicore/ml/scikitwrapper/ScikitLearnRULWrapper.java

```diff
@@ -0,0 +1,16 @@
+package ai.libs.jaicore.ml.scikitwrapper;
+
+import java.io.IOException;
+
+import org.api4.java.ai.ml.core.evaluation.IPrediction;
+import org.api4.java.ai.ml.core.evaluation.IPredictionBatch;
+
+import ai.libs.jaicore.ml.core.EScikitLearnProblemType;
+
+public class ScikitLearnRULWrapper<P extends IPrediction, B extends IPredictionBatch> extends ScikitLearnRegressionWrapper<P, B> {
```

Comment:
Is this now a wrapper for a pipeline taking time series data as an input and predicting a regression value? or just tabular data input and predicting a regression value? can we maybe connect the RUL wrapper to the regression wrapper as both do regression?

Reply:
It is actually a regression wrapper, but its working on time series data. I will rename it to TimeSeriesRegression to make it clear. In the field of PdM we could use the same wrapper for health index estimation, only the training labels are different in its interpretation